examples/guessing.rs

```diff
@@ -63,7 +63,7 @@ async fn main() -> Result<(), failure::Error> {
 
     let mut incoming = listener.incoming();
     while let Some(stream) = incoming.next().await {
-        runtime::spawn(play(stream?));
+        runtime::spawn(play(stream?)).await?;
```

Comment:
Isn't the whole point of `runtime` is lost when `.await`ing for every spawned future? `.await`ing results in sequential program: e.g. current version of guessing game does not support multiple simultaneous clients.